i18n/sample/src/uss/ZoweUSSNode.i18n.json

```diff
@@ -4,7 +4,7 @@
     "getChildren.error.response": "Retrieving response from ",
     "getChildren.responses.error.response": "The response from Zowe CLI was not successful",
     "getChildren.responses.open": "Open",
-    "deleteUSSNode.quickPickOption": "Are you sure you want to delete ",
+    "deleteUSSNode.quickPickOption": "Delete {0}? This will permanently remove it from your system.",
     "deleteUSSNode.showQuickPick.yes": "Yes",
```

Comment:
This should be Delete and Cancel as well to be consistent with the other choices